playground/src/Playground.tsx

```diff
@@ -1,13 +1,27 @@
-import React, { ReactNode } from 'react';
+import React, { CSSProperties } from 'react';
 import { LiveProvider, LiveEditor, LiveError, LivePreview } from 'react-live';
 import dracula from 'prism-react-renderer/themes/dracula';
 import toJSXString from 'react-element-to-jsx-string';
 
 const styles = {
   preview: {
     padding: '1rem',
-    border: '1px solid #e1e4e8',
-  },
+    border: '1px solid #E1E4E8',
+  } as CSSProperties,
+  editor: {
+    height: 'auto',
+    overflow: 'auto',
```

Comment:
I think it's not mandatory here, since there is no max-height or fixed height, I didn't checked.